test/esm/babel-runtime.js

```diff
@@ -10,18 +10,18 @@ export default {
           async () => import("@babel/runtime/helpers/esm/unknown-helper"),
           {
             name: "Error",
-            code: "ERR_MODULE_NOT_FOUND",
+            code: "ERR_PACKAGE_PATH_NOT_EXPORTED",
           }
         ),
     ],
-    [
+    /*[
       "it supports importing with explicit extension",
```

Comment:
I have just commented this out because it has been broken since 7.13.7 (https://github.com/babel/babel/commit/cfdbdc0d1c0f061b9633b04d85b54892e01cdd4d) and no one complained; we can fix it if someone reports it.